docs/ref/models/querysets.txt

```diff
@@ -2571,8 +2571,11 @@ because an implementation there isn't straightforward.
 
 The ``format`` parameter changes the output format from the databases's default,
 usually text-based. PostgreSQL supports ``'TEXT'``, ``'JSON'``, ``'YAML'``, and
-``'XML'``. MySQL supports ``'TEXT'`` (also called ``'TRADITIONAL'``) and
-``'JSON'``.
+``'XML'``. MariaDB and MySQL support ``'TEXT'`` (also called ``'TRADITIONAL'``)
+and ``'JSON'``. MySQL 8.0.16+ also supports ``'TREE'`` which is formatted
+similar to PostgreSQL's ``'TEXT'`` output and is used by default (if
+supported), to provide similarity between backends and more useful details as
+``TEXT`` on MySQL is very limited.
```

Comment:
* I think we need to add `'formats'` at the end of the lists.
* I'm not sure the very last clause adds much. 

So, suggested edit: 

```
The ``format`` parameter changes the output format from the databases's
default, usually text-based. PostgreSQL supports ``'TEXT'``, ``'JSON'``,
``'YAML'``, and ``'XML'`` formats. MariaDB and MySQL support ``'TEXT'`` (also
called ``'TRADITIONAL'``) and ``'JSON'`` formats. MySQL 8.0.16+ also supports
an improved ``'TREE'`` format, which is similar to PostgreSQL's ``'TEXT'``
output and is used by default, if supported.
```

Reply:
Not part of this diff but should that first sentence end `from the databases's
default,  which is usually text-based.`?